<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywallet</class>
 <widget class="QWidget" name="mywallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>904</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1278</width>
    <height>904</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1278</width>
    <height>904</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #10161f;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>0</y>
     <width>251</width>
     <height>904</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d9d8d4  ;</string>
   </property>
   <widget class="QPushButton" name="Mywallets_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: #c97940;
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>My wallets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Transmission_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Transmission</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>E:/Downloads/logo for a currency exchange named heliacal (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>-6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Dashboard_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 28pt &quot;Bangers&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Authentication_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: #4fee93;
font: 28pt &quot;Bangers&quot;;
letter-spacing: -1px;</string>
    </property>
    <property name="text">
     <string>Authentication</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">           border: none;  
color: #344b90;
background-color: #d9d8d4;
font: 22pt &quot;Rubik Wet Paint&quot;;
letter-spacing: -2px;
word-spacing: 3px;
</string>
    </property>
    <property name="text">
     <string>heliacal exchange</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="easyexchange_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: -1px;</string>
    </property>
    <property name="text">
     <string>Easy exchange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Profile_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currentprice_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Current price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Settings_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>850</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="market_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 28pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>market</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Dashboard_btn_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color:rgb(255, 255, 255);
font: 18pt &quot;Rubik Vinyl&quot;;
 border-radius: 10px;
background-color: rgb(16, 16, 16);</string>
   </property>
   <property name="text">
    <string>Create a wallet</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1011</width>
     <height>771</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
